integration-tests/paho/src/test/java/org/apache/camel/quarkus/component/paho/it/ActiveMQTestResource.java

```diff
@@ -38,7 +42,7 @@
             broker.setDataDirectory("target");
             broker.start();
         } catch (Exception e) {
-
+            LOGGER.error("Starting the ActiveMQ broker with exception.", e);
```

Comment:
Does it make any sense to continue with the execution of the tests if this fails? I wonder if throwing a RuntimeException would not be better, like they do e.g. here in Quarkus? https://github.com/quarkusio/quarkus/blob/master/integration-tests/artemis-core/src/test/java/io/quarkus/it/artemis/ArtemisTestResource.java#L23